CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift

```diff
@@ -284,15 +285,19 @@ public let ATTRIBUTE_NODES: [Node] = [
     children: [
       Child(
         name: "witnessMethodLabel",
-        kind: .token(choices: [.keyword(.witness_method)])
+        kind: .token(choices: [.keyword(.witness_method)]),
+        documentation: #"The "witnessMethod" label."#
```

Comment:
We generally prefer to use backticks `` ` `` instead of `"` to quote source code. It renders nicer as code font in docc.

Same below